src/unstract/sdk/adapters/vectordb/no_op/src/no_op_vectordb.py

```diff
@@ -0,0 +1,32 @@
+import time
+
+from llama_index.core.schema import TextNode
+from llama_index.core.vector_stores.types import VectorStore, VectorStoreQueryResult
+
+
+class NoOpVectorDB(VectorStore):
```

Comment:
@harini-venkataraman Better to rename this to `NoOpCustomVectorDB`.

Similarly we can do in other places.

Reply:
Done.